packages/fx-core/resource/strings.json

```diff
@@ -19,13 +19,31 @@
     "CancelProvision": "CancelProvision",
     "AlreadyProvisionNotice": "'%s' is already provisioned in the cloud, will create the teams app again if it has been removed.",
     "NotProvisionedNotice": "'%s' is not provisioned in the cloud yet. Please provision first.",
+
     "GenerateArmTemplateFailNotice": "'%s' Failed to create ARM template for required Azure services.",
-    "StartDeployArmTemplateNotice": "[%s] Start Deploying Arm templates.",
-    "SucessfullyCompileBicepNotice": "[%s] Successfully compile bicep files to JSON.",
-    "SucessfullyDeployArmTemplateNotice": "[%s] Successfully deploy Arm templates. Resource group name: %s. Deployment name: %s",
-    "FailedToDeployArmTemplateNotice": "[%s] Failed to deploy Arm templates. Resource group name: %s. Deployment name: %s",
-    "PollDeploymentStatusNotice": "[%s] Polling deployment status...",
-    "EndDeployArmTemplateNotice": "[%s] Successfully deployed Arm templates."
+
+    "DeployArmTemplates": {
+      "StartNotice": "[%s] Start deploying Arm templates.",
+      "CompileBicepSuccessNotice": "[%s] Successfully compile bicep files to JSON.",
+      "SuccessNotice": "[%s] Successfully deploy Arm templates. Resource group name: %s. Deployment name: %s",
+      "FailNotice": "[%s] Failed to deploy Arm templates. Resource group name: %s. Deployment name: %s",
+      "PollDeploymentStatusNotice": "[%s] Polling deployment status...",
+      "EndNotice": "[%s] End Arm templates deployment.",
```

Comment:
What's the different between "end" and “success"/"fail"?

Reply:
No matter success or fail, we will hit "end" event.
- Start
- - Success / fail
- End

Reply:
Since success/fail already expresses the progress has ended, maybe we no longer need to add an end log unless it brings extra information.

Reply:
Removed end notice.